crates/node/Cargo.toml

```diff
@@ -25,8 +25,6 @@ backtrace = "0.3"
 napi = {version = "2", default-features = false, features = ["napi3", "serde-json"]}
 napi-derive = {version = "2", default-features = false, features = ["type-def"]}
 path-clean = "0.1"
-proc-macro2 = "=1.0.32"
```

Comment:
Can you restore this? This is a hack for benchmark. (It uses different version of rustc because of nightly perf bug)